packages/py2_pip.rb

```diff
@@ -0,0 +1,33 @@
+require 'package'
+
+class Py2_pip < Package
```

Comment:
Won't this conflict with the pip installed with python3?

Reply:
I delete the only conflicting file. FileUtils.rm "#{CREW_DEST_PREFIX}/bin/pip"